src/components/Initialization/Initialization.js

```diff
@@ -37,7 +37,12 @@ class Initialization extends PureComponent<Props, State> {
     initializePath()
       .then(getNodeJsVersion)
       .then(nodeVersion => {
-        this.setState({ wasSuccessfullyInitialized: !!nodeVersion });
+        // the following dialog is just for debugging
+        dialog.showErrorBox('Path', window.process.env.PATH);

```

Comment:
Will remove this when it's ready to merge.

Reply:
Remove this and this is good to go :)